common/rabbitmq/templates/deployment.yaml

```diff
@@ -133,4 +138,9 @@ spec:
         - name: container-init
           configMap:
             name: {{ template "fullname" . }}-bin
+      {{- if $.Values.customConfig }}
+        - name: rabbitmq-custom-config
```

Comment:
Why is it named `rabbitmq-custom-config` and not `rabbitmq-etc` like the configmap? It might (at some point) contain more than the custom-config, right? And if not, why not name the configmap `…-custom-config`?

Reply:
it was because i thought it is butter to mount the hole conf.d but then changed it to only a mounted file instead, that to not override default config files 
naming schema set to ! rabbitmq-custom-config